scripts/gulpfiles/release_tasks.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Gulp script to build Blockly for Node & NPM.
+ */
+
+var execSync = require('child_process').execSync;
+var exec = require('child_process').exec;
+var fs = require('fs');
+var gulp = require('gulp');
+var readlineSync = require('readline-sync');
+var typings = require('./typings');
+
+var buildTasks = require('./build_tasks');
+var gitTasks = require('./git_tasks');
+var packageTasks = require('./package_tasks');
+
+const RELEASE_DIR = 'dist';
+
+// Gets the current major version.
+function getMajorVersion() {
+  delete require.cache[require.resolve('../../package.json')]
+  var { version } = require('../../package.json');
+  var re = new RegExp(/^(\d)./);
+  var match = re.exec(version);
+  if (!match[0]) {
+    return null;
+  }
+  console.log(match[0]);
+  return parseInt(match[0]);
+}
+
+// Updates the version depending on user input.
+function updateVersion(done, updateType) {
+  var majorVersion = getMajorVersion();
+  if (!majorVersion) {
+    done(new Error('Something went wrong when getting the major version number.'));
+  } else if (!updateType) {
+    // User selected to cancel.
+    done(new Error('Cancelling process.'));
+  }
+
+  switch (updateType.toLowerCase()) {
+    case 'major':
+      majorVersion++;
+      execSync(`npm --no-git-tag-version version ${majorVersion}.$(date +'%Y%m%d').0`, {stdio: 'inherit'});
+      done();
+      break;
+    case 'minor':
+      execSync(`npm --no-git-tag-version version ${majorVersion}.$(date +'%Y%m%d').0`, {stdio: 'inherit'});
+      done();
+      break;
+    case 'patch':
+      execSync(`npm --no-git-tag-version version patch`, {stdio: 'inherit'});
+      done();
+      break;
+    default:
+      done(new Error('In unexpected place in switch statement.'))
```

Comment:
Instead of "In inexpected place in switch", maybe something referencing "unexpected version update type" to make it clearer what caused the error.

Reply:
Done.